app/routes/company/index.tsx

```diff
@@ -1,23 +1,14 @@
-import { useRouteMatch, Route, Switch } from 'react-router-dom';
-import { CompatRoute } from 'react-router-dom-v5-compat';
+import { Route, Routes } from 'react-router-dom-v5-compat';
 import PageNotFound from '../pageNotFound';
 import CompaniesPage from './components/CompaniesPage';
 import CompanyDetail from './components/CompanyDetail';
 
-const CompanyRoute = () => {
-  const { path } = useRouteMatch();
-
-  return (
-    <Switch>
-      <CompatRoute exact path={path} component={CompaniesPage} />
-      <CompatRoute
-        exact
-        path={`${path}/:companyId`}
-        component={CompanyDetail}
-      />
-      <Route component={PageNotFound} />
-    </Switch>
-  );
-};
+const CompanyRoute = () => (
+  <Routes>
+    <Route path="" element={<CompaniesPage />} />
```

Comment:
The `index` you used for the other routes was prettier imo

Reply:
hehe agree - changed it in the other PR